lib/App/index.js

```diff
@@ -1521,17 +1521,28 @@ class App {
   }
 
   async createFlowJson() {
-    App.getManifest({ appPath: this.path });
+    if (App.hasHomeyCompose({ appPath: this.path }) === false) {
```

Comment:
why the strictEquals here?

Reply:
I always use strict equals with the very rare exception of checking `value == null` for checking both `undefined` and `null`. I personally dislike the `!` operator because it's easily missed while reading the code.